commcare_export/commcare_hq_client.py

```diff
@@ -26,6 +26,37 @@
 
 LATEST_KNOWN_VERSION='0.5'
 
+
+class CommcareHqClientBuilder(object):
```

Comment:
I think you could replace this class with two functions (eg `build_client_with_auth` and `build_client_with_session`), since it doesn't look like the builder instance is needed.  That would simplify the calls anyways.  This is a style question, so feel free to disregard if you prefer this approach.